### PR TITLE
Update pvoutput.markdown

### PR DESCRIPTION
**Pull request by rickylee64**
change to single quotes on value_template for "power-consumption", to removed error in HA file editor.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @rickylee64, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17295).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/pvoutput.markdown [hunk 1]
```diff
@@ -60,7 +60,7 @@ sensor:
   - platform: template
     sensors:
       power_consumption:
-              value_template: "{% if is_state_attr("sensor.pvoutput", "power_consumption", "NaN") %}0{% else %}{{ state_attr("sensor.pvoutput", "power_consumption") }}{% endif %}"
+              value_template: '{% if is_state_attr("sensor.pvoutput", "power_consumption", "NaN") %}0{% else %}{{ state_attr("sensor.pvoutput", "power_consumption") }}{% endif %}'
```
Comment (frenck) [inline source/_integrations/pvoutput.markdown]:
```suggestion
              value_template: "{% if is_state_attr('sensor.pvoutput', 'power_consumption', 'NaN') %}0{% else %}{{ state_attr('sensor.pvoutput', 'power_consumption') }}{% endif %}"
```

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide#quoting-style>

**Comment by homeassistant** [comment]
Hi @rickylee64, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17295).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @rickylee64 👍

**Pull request merged**